test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -0,0 +1,246 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/ossfuzz/protoToYul.h>
+#include <test/tools/ossfuzz/yulProto.pb.h>
+
+#include <ostream>
+#include <sstream>
+
+using namespace std;
+
+namespace yul
+{
+namespace test
+{
+namespace yul_fuzzer
+{
+ostream& operator<<(ostream& _os, BinaryOp const& _x);
```

Comment:
Could use a proper header for that. I am almost sure that you will need a class for that soon anyway to track state.

Reply:
done.